protocol/x/clob/keeper/liquidations.go

```diff
@@ -29,9 +29,21 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 ) error {
 	lib.AssertCheckTxMode(ctx)
 
+	pseudoRand := k.GetPseudoRand(ctx)
+
 	// Get the liquidation order for each subaccount.
+	// Process at-most `MaxLiquidationOrdersPerBlock` subaccounts, starting from a pseudorandom location
+	// in the slice.
 	liquidationOrdersForSorting := make([]types.LiquidationOrder, 0)
-	for _, subaccountId := range subaccountIds {
+	numSubaccounts := len(subaccountIds)
+	numLiqOrders := lib.Min(numSubaccounts, int(k.MaxLiquidationOrdersPerBlock))
+	indexOffset := 0
+	if numSubaccounts > 0 {
+		indexOffset = pseudoRand.Intn(numSubaccounts)
+	}
+	for i := 0; i < numLiqOrders; i++ {
```

Comment:
Is it worth measuring the latency of generating the liquidation orders? It seems like this would be helpful for determining how to tune `MaxLiquidationOrdersPerBlock` going forwards.

Or are we trying to limit metric usage due to how expensive it is?

Reply:
going to do metrics in a separate PR